### PR TITLE
test(rxMisc): Quickly convert text to dates

### PR DESCRIPTION
**Pull request by Droogans**
Replaces https://github.com/rackerlabs/encore-ui/pull/1144 so that visual regression is run.

### CODE REVIEW

**Comment by CITguy** [comment]
LGTM

File: src/rxMisc/rxMisc.page.js [hunk 1]
```diff
@@ -48,6 +48,38 @@ exports.rxMisc = {
     },
 
     /**
+       A convenience function to pass to `.then` when a date is needed from text.
+       If there is any manipulation of the text needed to render a valid date in
+       javascript, then you shouldn't use this function. If that is the case, point
+       please see http://rackerlabs.github.io/encore-ui/#/styleguide/basics#date-formatting.
```
Comment (Droogans) [inline src/rxMisc/rxMisc.page.js]: *point please see ...*

**Comment by parlarjb** [comment]
@Droogans No visual regression was run, I guess we don't have snapshots for this content?

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**